liquibase-standard/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -962,15 +971,15 @@ public void rollback(Database database, ChangeExecListener listener) throws Roll
             if (runInTransaction) {
                 database.commit();
             }
+            stopInstant = Instant.now();
             rollbackExecType = ExecType.EXECUTED;
             Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_OUTCOME, ExecType.EXECUTED.value.toLowerCase());
-            operationStopTime = new Date();
-            Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_OPERATION_STOP_TIME, Instant.ofEpochMilli(operationStopTime.getTime()).toString());
+            Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_OPERATION_STOP_TIME, stopInstant.toString());
             Scope.getCurrentScope().getLog(getClass()).fine("ChangeSet " + toString() + " has been successfully rolled back.");
```

Comment:
Hey @jasonlyle88, don't you think we should be logging execution time here if the execution of the generated rollback statements were successfully executed?

Reply:
Good catch @MalloD12. I added a line to log the rollback execution time. Let me know if this works for you. If it does, feel free to resolve this conversation!

Reply:
Nice, thanks @jasonlyle88!